Formula/termsql.rb

```diff
@@ -0,0 +1,21 @@
+class Termsql < Formula
+  desc "Convert text into SQL table and query it"
+  homepage "https://tobimensch.github.io/termsql/"
+  url "https://github.com/tobimensch/termsql.git", :using => :git
+  version "0.3"
+
+  depends_on :python => ["sqlite3", "sqlparse"]
+
+  def install
+    # Replace path to man pages in hard-coded setup.py
+    new_contents = File.read("setup.py").gsub(%r{\/usr\/share\/man\/man1\/}, man1)
```

Comment:
This can be done with `inreplace` instead, to avoid separate read/writes:

```ruby
inreplace "setup.py" do |s|
  s.gsub! %r{/usr/share/man/man1/}, man1
end
```

You should also report it upstream (build parameters never hurt :wink:)